README.rst

```diff
@@ -1,32 +1,25 @@
-DataStax Driver for Apache Cassandra
-====================================
+Scylla Python Driver
+====================
 
-.. image:: https://travis-ci.org/datastax/python-driver.png?branch=master
-   :target: https://travis-ci.org/datastax/python-driver
+A modern, feature-rich and highly-tunable Python client library for Scylla Open Source (2.1+) and Apache Cassandra (2.1+) and
+Scylla Enterprise (2018.1.x+) using exclusively Cassandra's binary protocol and Cassandra Query Language v3.
 
-A modern, `feature-rich <https://github.com/datastax/python-driver#features>`_ and highly-tunable Python client library for Apache Cassandra (2.1+) and
-DataStax Enterprise (4.7+) using exclusively Cassandra's binary protocol and Cassandra Query Language v3.
+The driver supports Python versions 2.7, 3.4, 3.5, 3.6, 3.7 and 3.8.
 
-The driver supports Python 2.7, 3.4, 3.5, 3.6, 3.7 and 3.8.
-
-**Note:** DataStax products do not support big-endian systems.
-
-Feedback Requested
-------------------
-**Help us focus our efforts!** Provide your input on the `Platform and Runtime Survey <https://docs.google.com/a/datastax.com/forms/d/10wkbKLqmqs91gvhFW5u43y60pg_geZDolVNrxfO5_48/viewform>`_ (we kept it short).
+.. **Note:** DataStax products do not support big-endian systems.
 
 Features
 --------
-* `Synchronous <http://datastax.github.io/python-driver/api/cassandra/cluster.html#cassandra.cluster.Session.execute>`_ and `Asynchronous <http://datastax.github.io/python-driver/api/cassandra/cluster.html#cassandra.cluster.Session.execute_async>`_ APIs
```

Comment:
don't we want to auto-generate our own version of the docs too ?
it's auto-generated from the code...

Reply:
in #10 we are generating the docs